org.testeditor.web.dropwizard.xtext/src/test/java/org/testeditor/web/dropwizard/xtext/integration/ExampleXtextApplicationIntegrationTest.xtend

```diff
@@ -60,5 +65,84 @@ class ExampleXtextApplicationIntegrationTest extends AbstractExampleIntegrationT
 		asJson.contains('"description":"MyFancyDescription"').assertTrue
 		asJson.contains('"source":').assertFalse
 	}
+	
+	
+	@Test
+	def void canRetrieveValidationMarkers() {
+		// given
+		val parent = remoteRepoTemporaryFolder.root;
+		val fileWithError = 'src/test/java/Broken.mydsl'
+		write(parent, fileWithError, 'Helo Typo!')
+		addAndCommit(remoteGit, fileWithError, 'add file with syntax error')
+		
+		// when
+		val response = createValidationMarkerRequest(fileWithError).submit.get
+
+		// then
+		response.status.assertEquals(OK.statusCode)
+		response.readEntity(String).assertEquals('[{"path":"src/test/java/Broken.mydsl","errors":1,"warnings":0,"infos":0}]')
+	}
+	
+	@Test
+	def void providesAllValidationMarkersOnUpdate() {
+		// given
+		val testStartTime = System.currentTimeMillis
+		val parent = remoteRepoTemporaryFolder.root;
+		val fileWithError = 'src/test/java/Broken.mydsl'
+		write(parent, fileWithError, 'Helo Typo!')
+		addAndCommit(remoteGit, fileWithError, 'add file with syntax error')
+		
+		// when
+		val response = createValidationMarkerUpdateRequest.submit.get
+
+		// then
+		response.status.assertEquals(OK.statusCode)
+		response.readEntity(String).assertEquals('[{"path":"src/test/java/Broken.mydsl","errors":1,"warnings":0,"infos":0}]')
+		response.headers.containsKey('lastAccessed').assertTrue
```

Comment:
lastAccessed is tested in multiple tests. should be put into one test and kept from others, what do you think?

Reply:
:+1: